lib/smooth/include/smooth/application/io/i2c/AxpRegisters.h

```diff
@@ -0,0 +1,142 @@
+/*
+Smooth - A C++ framework for embedded programming on top of Espressif's ESP-IDF
+Copyright 2019 Per Malmberg (https://gitbub.com/PerMalmberg)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+#pragma once
+
+#include <cstdint>
+#include <stdint.h>
+
+namespace smooth::application::sensor
+{
+    enum class Register : uint8_t
```

Comment:
`Register` is a very generic name, how about naming it `AxpRegister` to avoid future name clashes? Or, place the entire Axp code inside its own namespace?

Reply:
OK renamed to AxpRegister